auto3dseg/algorithm_templates/dints/configs/hyper_parameters.yaml

```diff
@@ -49,7 +49,7 @@ training:
   train_cache_rate: "$@training#cache_rate"
   validate_cache_rate: "$@training#cache_rate"
   transforms:
-    resample_to_spacing: "$@training#resample_to_spacing"
+    resample_resolution: "$@training#resample_resolution"
```

Comment:
The dollar sign is not necessary because there is no algorithmic operation
Example:
https://github.com/Project-MONAI/model-zoo/blob/8845f3a04766e254b156c23fc4d2f91659c056d4/models/spleen_deepedit_annotation/configs/train.json#L274C27-L274C47

Reply:
Thank you! All unnecessary $ signs in config files have been removed.